contrib/runners/orquesta_runner/tests/unit/test_with_items.py

```diff
@@ -580,12 +601,6 @@ def test_with_items_concurrency_pause_and_resume(self):
         self.assertEqual(t1_ex_db.status, wf_statuses.RUNNING)
         self.assertEqual(len(t1_ac_ex_dbs), concurrency)
 
-        # Reset the action executions to running status.
-        for ac_ex in t1_ac_ex_dbs:
-            self.set_execution_status(
-                ac_ex.liveaction["id"], action_constants.LIVEACTION_STATUS_RUNNING
-            )
-
         t1_ac_ex_dbs = ex_db_access.ActionExecution.query(
             task_execution=str(t1_ex_db.id)
         )
```

Comment:
Since we are not updating the action executions manually, this query is redundant.